low-level/files/opt/seagate/sspl/bin/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -110,124 +167,132 @@ def make_sspl_support_bundle_tarfile(parser, tmpFile):
             rmq_file.write("======================================================= \n")
         rmq_file.close()
 
-        if sspl_state == "active" and "-noencl" not in parser:
-            url = singleton_realstorencl.build_url("/")
-            COLLECTING_DEBUG_LOG_STARTED = False
-            for encl_trigger_log_retry_index in range(0, ENCL_TRIGGER_LOG_MAX_RETRY):
-                response = singleton_realstorencl.ws_request(url, singleton_realstorencl.ws.HTTP_POST, \
-                    post_data=f'/api/collectDebugData/"{SUPPORT_REQUESTOR_NAME}"{SUPPORT_EMAIL_ID}"{SUPPORT_CONTACT_NUMBER}')
+    def get_consul_data(self):
+        consul_sspl_cfg_cmd = f"/opt/seagate/{PRODUCT_FAMILY}/sspl/bin/consuldump.py -p sspl_config -l /tmp/ sspl/"
+        consul_sspl_cache_cmd = f"/opt/seagate/{PRODUCT_FAMILY}/sspl/bin/consuldump.py -p sspl_cache -l /tmp/ var/{PRODUCT_FAMILY}/sspl"
+        subout = self._run_command(consul_sspl_cfg_cmd)
+        subout = self._run_command(consul_sspl_cache_cmd)
+        del subout
+        # waiting to 2 to 3 second to genrate consul dump file.
+        time.sleep(3)
+        # Get latest consul sspl data from tmp directory.
+        all_subdirs = ['/tmp/'+d for d in os.listdir("/tmp/") if os.path.isdir('/tmp/'+d)]
+        self.consul_sspl_config = max([x for x in all_subdirs if x.startswith('/tmp/consuldump_sspl_config')], 
+                            key=os.path.getmtime)+'/sspl_'
+        self.consul_sspl_data = max([x for x in all_subdirs if x.startswith('/tmp/consuldump_sspl_cache')], 
+                            key=os.path.getmtime)+f'/var_{PRODUCT_FAMILY}_sspl'
+
+    def get_drives_smart_data(self):
+        lsscsi_cmd = "lsscsi | grep disk"
```

Comment:
this give drive list for both local server disks & enclosure disks. Need to get only local drives here, figure out a way, some unique characteristics

Reply:
Done

Reply:
I dont see this resolved yet, still see only "lsscsi | grep disk" being used.

Reply:
Yes, we don't need to remove this. we are filtering only local disk using below code.
https://github.com/Seagate/cortx-monitor/blob/37e72854cf69a6137bd03a8e6078d20fd61666aa/low-level/files/opt/seagate/sspl/bin/generate_sspl_bundle/sspl_bundle_generate#L201

I have tested this on multiple setup and it is giving result as expected (only local disk)

Reply:
Add a TODO here still, not full proof. For LDR R1 it would work

Reply:
Done